repose-aggregator/artifacts/build-system-packages.sh

```diff
@@ -55,3 +55,6 @@ sort --key=5 --unique > ${BUILD_DIR}/Files_RPM.out &&
 
 # Do the diff
 sdiff ${BUILD_DIR}/Files_DEB.out ${BUILD_DIR}/Files_RPM.out | grep -ve "^d.*<$"
+
+# Copy over the Vagrant files and fake services
```

Comment:
This script was originally used to compare the `.rpm`'s and `.deb`'s to ensure they were functionally the same. This came out of the building of them being managed completely separately. Now that the build is managed simultaneously in the Gradle builds, this would only serve as a sanity check. So that was a long way to a no, we probably don't need this script anymore.